_posts/2019-10-27-Merging-Multiple-SimpleCov-coverage-results.markdown

```diff
@@ -0,0 +1,73 @@
+---
+layout: post
+title: Merging-Multiple-SimpleCov-coverage-results
+date: 2019-10-29 13:56:00
+categories: ["Rails", "Simplecov", "Upgrades"]
+author: bronzdoc
+---
+
+As part of our Roadmap service at FastRuby.io, we have to analyze the test suite of the application we are upgrading to give a proper estimate on how long it will take us to upgrade. We use SimpleCov for this.
+We got in to the problem of having SimpleCov results across different containers for the same application. Our goal was to have a single result for the whole application, so in this blog post we are going to show you how we solved that problem.
```

Comment:
This sounds a little weird:

```
We got in to the problem of having SimpleCov results across different containers for the same application
```

Maybe something like this? 

```
Most of our clients use parallelization in their continuous integration tools. That means that SimpleCov generates multiple `.resultset.json` files for the same codebase.
```

Then you will have to continue with: 

```
Our goal was to have a single resultset file for the whole application, ... 
```